react-graph/src/PointNode.tsx

```diff
@@ -78,8 +78,8 @@ export class PointNode {
             return this.points;
         if (this.points != null && IncludeEdges !== undefined && IncludeEdges)
             return this.points.filter((pt,i) => (pt[0] >= Tstart && pt[0] <= Tend) || 
-                i < ((this.points?.length ?? 0) -1) && (this.points?[i+1][0] : 0) >= Tstart || 
-                i > 0  && (this.points?[i-1][0] : 0)<= Tend);
+                i < ((this.points?.length ?? 0) -1) && (this.points != null ?[i+1][0] : 0) >= Tstart || 
```

Comment:
This is not going to work `this.points?[i...` is not `if (this.points) {[i....` 
The `?` here is supposed to catch cases where `this.point == null` and stop the rest of the statement but if `this.point` is not null it should read `this.point[i+1]....` with your change that turns into `true[i+1]`